modules/core/client/services/angular-compat.js

```diff
@@ -0,0 +1,13 @@
+import angular from 'angular';
+
+function get(name) {
+  return angular.element(document).injector().get(name);
+}
+
+export function $broadcast(...args) {
+  return get('$rootScope').$broadcast(...args);
+}
+
+export function eventTrack(...args) {
+  return get('$analytics').eventTrack(...args);
+}
```

Comment:
If there started to be a lot of stuff in here it could be refactored to have one file per "angular-thing" then used something like:

```js
import { eventTrack } from '@/modules/core/client/angular-compat/$analytics'
```
```